server/routes/albums.js

```diff
@@ -0,0 +1,43 @@
+/*
+Server Albums Route | HOLDING Web App
+GROUP 3: Douglas MacKrell, Briahana Mauge, Joseph P. Pasaoa, Kathy Puma
+*/
+
+
+/* HELPERS */
+const log = console.log;
+
+
+/* MODULE INITS */
+const express = require('express');
+    const router = express.Router();
+    router.use(express.json());
+    router.use(express.urlencoded({extended: false}));
+// Database 
+const db = require('../db.js');
+
+
+/* MIDDLEWARE */
+const middleWare = async (req, res, next) => {
+  try {
+    let response = await db.any("SELECT * FROM users;");
+    res.json({
+        status: "success",
+        message: req.get('host') + req.originalUrl,
+        payload: response
```

Comment:
Change payload to body.

Reply:
okay, will do

Reply:
done: payloads changed to body on all routes